migrator/migrator.go

```diff
@@ -37,18 +37,25 @@ import (
 
 // PouchMigrator is a tool to migrate docker containers to pouch containers
 type PouchMigrator struct {
-	debug         bool
-	containerd    *ctrd.Ctrd
+	debug  bool
```

Comment:
LGTM, unless you add more description of the fields of this most important struct `PouchMigrator `.